src/docs/user/ProActiveUserGuide.adoc

```diff
@@ -2611,6 +2611,72 @@ Task Synchronization API can be used for multiple dynamic synchronization patter
 
 A complete example of Producer / Consumer, _ServiceDeployerConsumer_ can be downloaded link:examples/ServiceDeployerConsumer.xml[here].
 
+[[_task_signal_api]]
+=== Signal API
+
+The *Signal API* is a high-level API that enables users to interact with running jobs via signals (we also call them actions).
+That is, it allows a job to expose specific actions (e.g., soft stop, scale up, scale down) that can triggered by users from the *Workflow Execution portal*.
+
+More specifically, the Signal API allows a workflow to:
+
+  . expose signals, i.e., display them in the Workflow Execution portal (e.g., display the `soft_stop` signal)
+
+  . wait to receive these signals, or check for their reception regularly, and
+
+  . execute the corresponding behavior (defined in the workflow) at the signal reception.
+
+The Signal API uses the synchronization API to provide a *persistent key-value channel* to store job signals.
+It uses a single channel (called *PA_SIGNALS_CHANNEL*) to store the signals of all jobs. The signal channel contains an entry per job.
+The key of each entry is the job id, whereas the value of the entry is the set of signals used by the considered job.
+Job signals are organized in a link:https://docs.oracle.com/javase/8/docs/api/java/util/Set.html[Java Set] structure,
+which means that a signal appears only once in the set, even if it is sent many times.
+
+As already mentioned, the Signal API is based on the synchronization API. However, these APIs are intended for different purposes.
+The Synchronization API is used to exchange information (e.g., messages) between tasks and between jobs, whereas the
```

Comment:
Signal API -> *Signal API* 
Synchronization API -> *Synchronization API*

Reply:
between tasks and between jobs -> between tasks and jobs

Reply:
I used the same term in the synchronisation API description :)